src/components/molecule_3d.jsx

```diff
@@ -162,6 +172,20 @@ class Molecule3d extends React.Component {
       Molecule3d.render3dMolModel(glviewer, this.props.modelData);
     }
 
+    if (this.props.outlineWidth !== this.lastOutline.width ||
+        this.props.outlineColor !== this.lastOutline.color) {
+      if (this.props.outlineWidth) {
+        this.lastOutline = {
+          style: 'outline',
+          width: this.props.outlineWidth,
+          color: this.props.outlineColor,
+        };
+      } else {
+        this.lastOutline = {};
```

Comment:
Should this be `{ width: 0.0 }`?  Or maybe the initialization in the constructor should be `{}`.